bin/changelog/README.md

```diff
@@ -0,0 +1,55 @@
+# Changelog Script
+
+This folder contains the logic for a changelog script that can be used for generating changelog entries from either pull requests added to a Github milestone, or pull requests that are part of a Zenhub release.
+
+## Usage:
+
+You can implement the script in you `package.json` in the simplest form by adding the following to the `"scripts"` property (assuming it is installed in `./bin`):
+
+```json
+{
+  "scripts": {
+    "changelog": "node ./bin/changelog",
+  }
+}
+```
+
+## Configuration
+
+The following configuration options can be set for the changelog script. **Note:** you can use all of these options but environment variables overwrite `package.json` config and command line arguments overwrite environment variables.
```

Comment:
related to my previous comment, do we want 3 sources of configuration? Wouldn’t env and package be enough for most cases, one will act as the default stand (package.json) and one would be quick changes (env)

Reply:
If we drop to two (which I'm not opposed to) I think it should be `package.json` and cli.  However, we'd still need to keep `env` for the api tokens.  One thing I don't like about the current `env` variables is I think they are a bit too generic and there's risk of bleed from other apps. So I think I'll go ahead and remove all env variables _except_ the api tokens.

Reply:
Implemented in 3500d57, 79733fd, and bbe101b